projects/packages/connection/changelog/update-connection-processing-whole-input-comments

```diff
@@ -0,0 +1,5 @@
+Significance: patch
```

Comment:
Technically this should be major, since the signature of the public method `Automattic\Jetpack\Connection\Manager->setup_xmlrpc_handlers()` is being changed in a non-compatible way.

But if you make a case that the method really should have been private even though it was public (and not marked `@private` or `@internal` or anything like that), 🤷.

Reply:
Updated to patch again after @sergeymitr 's suggestion to deprecate `$request_params` arg in `setup_xmlrpc_handlers`